### ISSUE TITLE
deployment example 

### ISSUE THREAD

**Issue by jciturras** [whiskyechobravo/kerkoapp#24]
Thanks a lot for the example site. But I have been trying to deploy an app using heroku, but I failed.

¿How did you deploy this app to a Heroku free server?

**Comment by davidlesieur** [comment]
Unfortunately, I have never used Heroku. Can you provide more details on the problems encountered? Any error messages?

**Comment by jciturras** [comment]
Yeah...it's a particular problem, indeed. But my problem isn't with the platform in specific. I would be pleased to know how you handled deploying the app on a web server ¿What would you recommend for those purposes? 

Thanks a lot for your response!

**Comment by davidlesieur** [comment]
Have you seen [this guide](https://gist.github.com/davidlesieur/e1dafd09636a4bb333ad360e4b2c5d6d) on deploying on Ubuntu? (It's mentioned in KerkoApp's README, but maybe easy to miss)
